RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSeedProducer.h

```diff
@@ -0,0 +1,131 @@
+#ifndef RecoEgamma_EgammaElectronProducers_LowPtGsfElectronSeedProducer_h
+#define RecoEgamma_EgammaElectronProducers_LowPtGsfElectronSeedProducer_h
+
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/EcalRecHit/interface/EcalRecHitCollections.h"
+#include "DataFormats/EgammaReco/interface/ElectronSeed.h"
+#include "DataFormats/EgammaReco/interface/ElectronSeedFwd.h"
+#include "DataFormats/ParticleFlowReco/interface/PFCluster.h"
+#include "DataFormats/ParticleFlowReco/interface/PFClusterFwd.h"
+#include "DataFormats/ParticleFlowReco/interface/PFRecTrack.h"
+#include "DataFormats/ParticleFlowReco/interface/PFRecTrackFwd.h"
+#include "DataFormats/ParticleFlowReco/interface/PreId.h"
+#include "DataFormats/ParticleFlowReco/interface/PreIdFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "MagneticField/Engine/interface/MagneticField.h"
+#include "RecoEcal/EgammaCoreTools/interface/EcalClusterLazyTools.h"
+#include "RecoEgamma/EgammaElectronProducers/interface/LowPtGsfElectronSeedHeavyObjectCache.h"
+
+class LowPtGsfElectronSeedProducer final : public edm::stream::EDProducer< edm::GlobalCache<lowptgsfeleseed::HeavyObjectCache> >
+{
+  
+ public:
+  
+  explicit LowPtGsfElectronSeedProducer( const edm::ParameterSet&, 
+					 const lowptgsfeleseed::HeavyObjectCache* );
+
+  ~LowPtGsfElectronSeedProducer() override;
+  
+  static std::unique_ptr<lowptgsfeleseed::HeavyObjectCache> 
+    initializeGlobalCache( const edm::ParameterSet& conf ) {
+    return std::make_unique<lowptgsfeleseed::HeavyObjectCache>(lowptgsfeleseed::HeavyObjectCache(conf));
+  }
+  
+  static void globalEndJob( lowptgsfeleseed::HeavyObjectCache const* ) {}
+  
+  void beginLuminosityBlock( edm::LuminosityBlock const&, 
+			     edm::EventSetup const& ) override;
+
+  void produce( edm::Event&, const edm::EventSetup& ) override;
+  
+  static void fillDescription( edm::ParameterSetDescription& );
```

Comment:
The proper signature is
```suggestion
  static void fillDescriptions( edm::ConfigurationDescriptions& );
```

Reply:
Done, [here](https://github.com/cms-sw/cmssw/pull/25455/commits/4ac0635875a96f1e10fc7e72650f72acfece1085)